modules/datadict/test/datadictTest.php

```diff
@@ -161,11 +151,6 @@ function testDataDictSearchKeywordFiltersWithoutData()
         );
 
         $searchKey[0]->sendKeys("noExist");
-        $searchButton = $this->webDriver->findElement(
-            WebDriverBy::Name("filter")
-        );
-        //search exist data
-        $searchButton->click();
 
         $res = $this->webDriver->executescript(
```

Comment:
Remove line 155-159, 
insert this [
        $res = $this->webDriver->executescript(
            "return document.querySelector".
                  "('#lorisworkspace > div > div > div.panel.panel-default >".
                  "div.table-header.panel-heading > div > div').textContent"
        );
        $this->assertContains("0 rows displayed", $res);
] instead of the old.